packages/functional-tests/pages/login.ts

```diff
@@ -203,7 +203,7 @@ export class LoginPage extends BaseLayout {
 
   async isSignUpCodeHeader() {
     const header = this.page.locator(selectors.SIGN_UP_CODE_HEADER);
-    header.waitFor({ state: 'visible' });
+    await header.waitFor({ state: 'visible' });
```

Comment:
this missing await was consistently causing at least one test to fail.  I guess we've been on the lucky side of a race condition up 'til this PR?

Reply:
Not sure.  I do see `isVisible` used elsewhere in our Playwright tests.